src/services/external/excalidraw-backend/excalidraw.server.ts

```diff
@@ -60,6 +62,8 @@ import {
   minCollaboratorsInRoom,
   resetCollaboratorModeDebounceWait,
 } from './types/defaults';
+import { ActivityAdapter } from '@services/adapters/activity-adapter/activity.adapter';
+import { AgentInfo } from '@core/authentication';
```

Comment:
please move these absolute paths upper, so the relative (closer to the domain) are lower.
Try to stick to that rule in the future.